packages/notification/tsconfig.json

```diff
@@ -7,6 +7,7 @@
     "outDir": "lib",
     "lib": ["ES2021", "DOM"],
     "strict": true,
+    "useUnknownInCatchVariables": false,
```

Comment:
The proper way to handle a catch block error in TypeScript is to check the error type and return a suitable error message manually. You can create a function and apply it to all catch block errors, as described in [this article](https://kentcdodds.com/blog/get-a-catch-block-error-message-with-typescript).

Reply:
+1 for this.